spec/moonshot/commands/build_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe Moonshot::Commands::Build do
+
+  it 'should not handle --skip-ci-status and build correctly' do
+    soso = Moonshot::CommandLineDispatcher.new('build', subject, {})
```

Comment:
Can you please change `soso` to something more self explanatory?

Reply:
Fixed